### PR TITLE
Add smiley face to Happy Collaborating message

### PR DESCRIPTION
**Pull request by aryanfaghihi**
How can one be happy collaborating without seeing a smiley face? 

This PR simply adds that smiley face to make your day :)

### CODE REVIEW

File: mob.go [hunk 1]
```diff
@@ -702,7 +702,7 @@ func execute(command string, parameter []string, configuration Configuration) {
 		} else if configuration.Timer != "" {
 			startTimer(configuration.Timer, configuration)
 		} else {
-			sayInfo("It's now " + currentTime() + ". Happy collaborating!")
+			sayInfo("It's now " + currentTime() + ". Happy collaborating! :)")
```
Comment (ngocdaothanh) [inline mob.go:705]: Should we use the Unicode one?
https://emojipedia.org/smiling-face-with-smiling-eyes/

Comment (aryanfaghihi) [reply mob.go:705]: I kind of like the subtle nature of ":)" but open to using a Unicode one too :)

**Comment by simonharrer** [comment]
Great PR :)

Because it is so great, we need a changelog entry. Can you add that, please?

File: mob.go (hunk 1 above)
Comment (simonharrer) [reply mob.go:705]: I would go with the subtle nature for now.

**Comment by aryanfaghihi** [comment]
@simonharrer No worries, done!

**Pull request merged**